### ISSUE TITLE
Gamescope Steam Session: no longer able to connect to any network

### ISSUE THREAD

**Issue by dklementowski** [ValveSoftware/steam-for-linux#11518]
#### Your system information

* Steam client version (build number or date): 1731433018
* Distribution (e.g. Ubuntu): Arch
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: 
[steam-logs.tar.gz](https://github.com/user-attachments/files/17925936/steam-logs.tar.gz)

* GPU: AMD

#### Please describe your issue in as much detail as possible:
I use Steam on Arch with [gamescope-session-steam](https://aur.archlinux.org/packages/gamescope-session-steam-git) for SteamOS-like experience on my HTPC. Its integration with NetworkManager started malfunction after today's client update and I'm no longer able to connect to any network, even though I see them being listed and I can also disconnect. Wifi network ask for password and then fails immediately, but same it cannot connect to a wired network. Using nmtui from tty I can connect to any network and then Steam session proceeds to work without any issue.

I can also reproduce the thing inside Plasma desktop using nested gamescope session initiated like bellow.

#### Steps for reproducing this issue:

1. Run standalone gamescope-steam session or nested with command
```
gamescope -w 1920 -h 1080 -f --steam -- steam -gamepadui -steamos
```
2. Disconnect from any network, works 
3. Try to connect to any wired or wireless network, it doesn't work

Maybe there's some new requirement and I'm missing some package? Maybe it's a general bug? I think I haven't installed any updates, just the Steam client updated itself.

**Comment by Uklosk** [comment]
Do you have lib32-libnm installed?

**Comment by dklementowski** [comment]
Well, I didn't. Apparently Steam previously shipped its own or smh? I’ll confirm later and let them know on AUR that this should now be dependency, thanks.

**Comment by dklementowski** [comment]
Yep, now it works. I think Arch Linux's Steam package from multilib should include that as dependency by default, will report. I think this issue can be closed for now.
